api/request.go

```diff
@@ -3,6 +3,9 @@ package api
 type VolumeMountRequest struct {
 	VolumeName string
 	MountPoint string
+	ReadWrite  string // either "rw or "ro"
```

Comment:
Please update documents, including cli_reference and all related drivers. Especially the doc need to be updated for which option available for which driver.